src/chat/channel.py

```diff
@@ -259,7 +264,7 @@ def printLine(self, chname, text, scroll_forced=False, formatter=Formatters.FORM
             # Fallback and ask the client. We have no Idea who this is.
             color = self.chat_widget.client.player_colors.getUserColor(player.id)
 
-        if mentioned:
+        if mentioned and name_of_author_message != self.chat_widget.client.login:
```

Comment:
I'd rather we used the 'User' class for nick comparison ('me' member in client, represents the local faf user, has a 'player' member that's not None when we're logged in).

Reply:
It's serviceable (given that with current mess of a code it's annoying to try and pass 'me' in chatter constructor), but check for me.player being None (might be the case if we lose connection to the server, but keep it to chat). Using some local variable for 'me' should make things a bit shorter.

Reply:
Ok, I added a new variable for it.